app/graphql/types/inputs/ont/libraries_input_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  module Inputs
+    module Ont
+      # The input arguments for Ont libraries.
+      class LibrariesInputType < BaseInputObject
```

Comment:
I think this name is a bit misleading since it infers you're providing information about a library, when you're actually specifying a plate and grouping strategy.  Maybe this should be called something like `LibraryCreationArgumentsInputType`.

Reply:
Input type renamed